unittests/test_schedulers.py

```diff
@@ -530,6 +530,52 @@ def test_submit_timelimit(self):
         pytest.skip("PBS minimum time limit is 60s")
 
 
+class TestTorqueJob(_TestJob, unittest.TestCase):
+    @property
+    def sched_name(self):
+        return 'torque'
+
+    @property
+    def launcher_name(self):
+        return 'local'
```

Comment:
You may remove this completely if you inherit from `TestPbsJob` as I suggest below.

Reply:
Also in order to make sure that the unit tests pass for the Torque backend, you should run them as follows:

```
./test_reframe.py --rfm-user-config=/path/to/config/using/torque.py
```

Reply:
i've tested it with a user config and all torque tests pass (test_submit_timelimit skipped).

Reply:
Yep, I've just adapted our PBS config and works fine also with Slurm wrappers. That's good news, because we can CI regularly this backend. I will push the CI changes to this PR as well.